lib/client/api.go

```diff
@@ -540,7 +542,9 @@ func RetryWithRelogin(ctx context.Context, tc *TeleportClient, fn func() error)
 	}
 	log.Debugf("Activating relogin on %v.", err)
 
-	key, err := tc.Login(ctx)
+	// TODO(codingllama): Allow for the passwordless flag to drip down here?
+	//  We'd have to change it to a global flag, but that seems alright.
+	key, err := tc.Login(ctx, false /* pwdless */)
```

Comment:
I think we should support passwordless consistently, including here. I wonder if we can make `pwdless bool` a property of `TeleportClient` instead?

Reply:
Sounds fair, given that similar fields are part of the config. Done.